### DISCUSSION TITLE
[BUG ?] Readme doesn't display

### DISCUSSION THREAD

**Discussion by MidlajYH** [community/community#35443] [Mobile]
https://user-images.githubusercontent.com/85442225/194577821-b80df381-2ecc-4510-8a45-4e8a596a4c40.mp4

Tried both in Dark & Light theme, it doesn't show

In browser it's working fine.

https://user-images.githubusercontent.com/85442225/194577990-bb190afe-a090-4a70-baa4-af8539799790.mp4

Repo: https://github.com/sudheerj/javascript-interview-questions

My Device: Poco X3 Pro
Android Version: 12

Using Github Mobile Latest Beta 1.84.0-beta (10090)

**Comment by joshcangit** [comment]
I think it's just some repositories such as that one.
Can't even load the Readme.

Readme in other repositories get truncated at the top half of the title.
They have to somehow be reloaded.
